frontend/frames/buttons.py

```diff
@@ -0,0 +1,36 @@
+from tkinter import Button, font, Frame, PhotoImage
+
+class ButtonFrame(Frame):
+    '''Frame with buttons'''
+    def __init__(self, window, base, **kwargs):
+        Frame.__init__(self, window, **kwargs)
+        self.base = base
+        self.wheel = PhotoImage(file="./frontend/frames/wheel.png")
```

Comment:
It seems that PhotoImage can't be placed before __init__.:
```
Traceback (most recent call last):
  File "main.py", line 3, in <module>
    from frontend.mainwindow import MainWindow
  File "/home/rafal/code/alinka/frontend/mainwindow.py", line 21, in <module>
    from frontend.frames.buttons import ButtonFrame
  File "/home/rafal/code/alinka/frontend/frames/buttons.py", line 4, in <module>
    class ButtonFrame(Frame):
  File "/home/rafal/code/alinka/frontend/frames/buttons.py", line 6, in ButtonFrame
    wheel = PhotoImage(file="./frontend/frames/wheel.png")
  File "/usr/lib/python3.6/tkinter/__init__.py", line 3545, in __init__
    Image.__init__(self, 'photo', name, cnf, master, **kw)
  File "/usr/lib/python3.6/tkinter/__init__.py", line 3489, in __init__
    raise RuntimeError('Too early to create image')
RuntimeError: Too early to create image
```
But I made some changes to remove images from class attributes. I needed them as a class attributes because:
```
Note: When a PhotoImage object is garbage-collected by Python (e.g. when you return 
from a function which stored an image in a local variable), the image is cleared even
 if it’s being displayed by a Tkinter widget.

To avoid this, the program must keep an extra reference to the image object. 
A simple way to do this is to assign the image to a widget attribute, like this:

label = Label(image=photo)
label.image = photo # keep a reference!
label.pack()
```
the same effect give put image as a class attribute.